Header header
float32 acceleration
float32 acceleration_averaged

